docs/building-experiments/experiments/experiment-status.md

```diff
@@ -16,7 +16,7 @@ If you go on to the details page for an experiment and hover over the status tag
 ## Requirements to Move to the Next Experiment Status
 | Status | Non-Eppo Randomized Experiment | Eppo Randomized Experiment |
 | :--- | :--- | :--- |
-| `DRAFT` | Go to `RUNNING` if… <li> Experiment key added </li> <li> At least 1 decision metric added </li>  <li> Variation added </li> <li> Experiment started <em>(the assignment start date passed)</em> </li>| Go to `RUNNING` if… <br> <li> Experiment key added </li> <li> At least 1 decision metric added </li>  <li> Variation added </li> <li> Experiment started <em>("Start Experiment" button pressed)</em> </li>|
-| `RUNNING` | Go to `WRAP UP` if… <li> Experiment ended <em>(Larger of the end dates entered by the user has passed)</em> </li> <br> Go to `COMPLETED` if… <li> Decision made <em>(user clicks the “Make Decision” button and fills out the outcome and decision)</em> - This will update the end date if going from `RUNNING` to `COMPLETED` </li> | Go to `WRAP UP` if… <li>  Experiment ended <em>(User stops assignments & event analysis date has passed)</em> </li> <br> Go to `COMPLETED` if… <li> Decision made <em>(user clicks the “Make Decision” button and fills out the outcome and decision)</em> - This will stop assignments if still on and update the end date if going from `RUNNING` to `COMPLETED` </li> <br> Uncommon situation: If the event analysis end date is set to run longer than when assignments are stopped, clicking “Stop Assignment” will not change the experiment status from `RUNNING`. The experiment will remain in the `RUNNING` status until the larger of the two end dates have passed. Once that date has passed, the experiment will move into the `WRAP UP` status. |
-| `WRAP UP` | Note: This step is skipped if user clicks “Make Decision” from the `RUNNING` state <br><br> Go to `COMPLETED` if… <li> Decision made <em>(user clicks the “Make Decision” button and fills out the outcome and decision)</em> - the dates are not affected </li> | Note: This step is skipped if user clicks “Make Decision” from the `RUNNING` state <br><br> Go to `COMPLETED` if… <li> Decision made <em>(user clicks the “Make Decision” button and fills out the outcome and decision)</em> - the dates are not affected </li> |
+| `DRAFT` | Go to `RUNNING` if… <li> Experiment key added </li> <li> At least 1 decision metric added </li>  <li> Variation added </li> <li> Experiment started <em>(the assignment start date passed)</em> </li>| Go to `RUNNING` if… <br/> <li> Experiment key added </li> <li> At least 1 decision metric added </li>  <li> Variation added </li> <li> Experiment started <em>("Start Experiment" button pressed)</em> </li>|
```

Comment:
There was a syntax error in this file with the `<br>` tag that should be `<br />`